scargo/file_generators/mock_gen.py

```diff
@@ -53,11 +49,26 @@ def generate_mocks(src_header: Path, src_dir: str) -> bool:
     header_data = parse_file(src_header)
 
     # generate public interface header
-    gen_header(dst_header, "class_interface.h.j2", header_data)
+    create_file_from_template(
```

Comment:
create_file_from_template are repeated a few times and differ only in first param. maybe better to make list of j2 files and use it in the loop?

Reply:
They differ in first and second param. I can add a loop, but for two params it's not as beneficial, and it's as likely that a change will be needed for only one of the templates.